network/src/protocols/discovery/test.rs

```diff
@@ -108,8 +111,8 @@ async fn expect_address_update(
 
 fn generate_network_pub_keys_and_signer() -> (NetworkPublicKeys, Ed25519PrivateKey) {
     let mut rng = StdRng::from_seed(TEST_SEED);
-    let signing_priv_key = Ed25519PrivateKey::generate_for_testing(&mut rng);
-    let (_, identity_pub_key) = x25519::compat::generate_keypair(&mut rng);
+    let signing_priv_key = Ed25519PrivateKey::generate(&mut rng);
```

Comment:
`generate_for_testing`

Reply:
This one is a little nuanced but the test shouldn't depend on the uniqueness of keys.. but we do generate secondary EC key afterward.